package.json

```diff
@@ -341,7 +341,7 @@
     "@metamask/phishing-controller": "^15.0.0",
     "@metamask/post-message-stream": "^10.0.0",
     "@metamask/ppom-validator": "0.39.0",
-    "@metamask/preinstalled-example-snap": "^0.7.1",
+    "@metamask/preinstalled-example-snap": "^0.7.2",
```

Comment:
We should be able to remove these now: https://github.com/MetaMask/metamask-extension/blob/d788715a8c44c14139f3fa4cb1b165f500d0f84d/test/e2e/flask/snaps/preinstalled-example.spec.ts#L69-L70